README.md

```diff
@@ -44,16 +44,21 @@ For more details, see [building from source](https://github.com/Xpra-org/xpra/tr
 ---
 
 # Usage
+## Initial requirements
+`xpra` must be installed on the client and the host (version matters).  
```

Comment:
Please remove "(version matters)" or make it a leak to the supported versions wiki page.